McZapkie/hamulce.cpp

```diff
@@ -118,7 +118,6 @@ double PFVd( double PH, double PL, double const S, double LIM, double const DP )
         LIM = LIM + 1;
         PH = PH + 1.0; // wyzsze cisnienie absolutne
         PL = PL + 1.0; // nizsze cisnienie absolutne
-        assert( PH != PL );
         double sg = PL / PH; // bezwymiarowy stosunek cisnien
```

Comment:
Never mind my previous comments, I was confused.

sg should probably be clamped at 1.0 like in PFVa.

Reply:
I capped it the same way it's done in PFVa.